Gemfile

```diff
@@ -25,91 +25,93 @@ def location_for(place_or_version, fake_version = nil)
   end
 end
 
+# Gems needed by other groups
+gem 'rake', :require => false
+
 group :development do
-  gem 'rake',                         :require => false
```

Comment:
It appears this is the only change? 

Usually development is installed with all groups. What cases are you running into that you want to exclude the development group?